packages/compass-import-export/src/import/analyze-csv-fields.spec.ts

```diff
@@ -88,6 +88,7 @@ describe('analyzeCSVFields', function () {
         'objectId',
         'regex',
         'timestamp',
+        'symbol',
```

Comment:
I ran into the same thing when I tried replacing the CSV parser yesterday: I have no idea how this passed before because we can't detect symbols as a type. The file is basically even empty and it should have been skipped all along.